### PR TITLE
feat: Add local storage labels to kube_persistentvolume_info

### PR DESCRIPTION
**Pull request by nabokihms**
Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>

**What this PR does / why we need it**:
Local volumes in Kubernetes differ from other volumes, e.g., they do not have the max cap as other persistent volumes. Instead, local PVs can consume all the available space on the disk. Thus, there should be a way to distinguish local PVs (and other rules to monitor them). 

**How does this change affect the cardinality of KSM**: *(increases, decreases or does not change cardinality)*
Yes, it make local storage volumes metrics looks like `kube_persistentvolume_info{local_path="/mnt/data", local_fs="ext4"}`

### CODE REVIEW

File: docs/persistentvolume-metrics.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-| kube_persistentvolume_created | Gauge | Unix Creation Timestamp | seconds | `persistentvolume`=&lt;persistentvolume-name&gt; <br> | EXPERIMENTAL |
+| Metric name | Metric type | Description | Unit (where applicable) | Labels/tags                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Status |
```
Comment (dgrisonnet) [inline docs/persistentvolume-metrics.md:3]: would you mind reverting the changes made to this file and just add the two labels that you added to the `kube_persistentvolume_info` metric?

Comment (nabokihms) [reply docs/persistentvolume-metrics.md:3]: Oh, sorry. Yeah, it is some kind of an autoformatting in my IDE.

**Comment by nabokihms** [comment]
The idea to add specific labels to a dedicated metric seems more than ok to me.

Something like:
```
kube_persistentvolume_info{type="csi", persistentvolume="test-1"}
kube_persistentvolume_csi_info{driver="ABC", volume_handle="csi-volume-handle"}
kube_persistentvolume_info{type="local", persistentvolume="test-1"}
kube_persistentvolume_local_info{path="/mnt", fs="ext4"}
```

@dgrisonnet  Does it seem ok to you? If so, I can dig to implement it more generic way.

**Comment by nabokihms** [comment]
I reverted table changes and add host path labels.

**Comment by nabokihms** [comment]
Related https://github.com/dexidp/dex/pull/2562

**Comment by dgrisonnet** [comment]
LGTM, thank you @nabokihms for the contribution :)

/lgtm
/approve

**Comment by dgrisonnet** [comment]
Yeah your suggestion is pretty similar to what I had in mind, I'll start a github discussion around that topic.

**Comment by nabokihms** [comment]
@dgrisonnet I'd like to participate, if it is possible.
😄

**Pull request merged**